core/schema/platform.graphqls

```diff
@@ -1,12 +1,11 @@
 """
 The platform config OS and architecture in a Container.
-The format is [os]/[platform]/[version] (e.g. darwin/arm64/v7, windows/amd64, linux/arm64).
+
+The format is [os]/[platform]/[version] (e.g. "darwin/arm64/v7", "windows/amd64", "linux/arm64").
 """
 scalar Platform
 
 extend type Query {
-  """
-  The default platform of the builder.
-  """
+  "The default platform of the builder."
```

Comment:
get back to `"""`

Reply:
Why? It's a single line comment?

Reply:
*https://github.com/dagger/dagger/pull/4367#issuecomment-1398350547*
> For one liners you can still use the triple quotes. Instead of doing it only when needed, I suggest making it a convention. It also helps with git diffs in case the content changes (e.g., turning it multiline or needing double quotes in an example).

So if we create some parser, it's always the same rule, we will parse a `"""` until the netxt `"""`. And for git diff, if we decide to add more description, the diff will just be in between those `"""`, and not switching from `"` to `"""`. So you can focus on what actually changed

```diff
  """
  Provides super interesting information
+ 
+ With this extra super information
+ about it.
  """
  ```
  
  Instead of 
```diff
- "Provides super interesting information"
+ """
+ Provides super interesting information
+ 
+ With this extra super information
+ about it.
+ """
  ```
  ```diff

Reply:
But I won't block for that, I would do some PR dedicated to change every description into the `"""` so we can merge this one (which brings more goodness than bare `"""` ).